docs/io.cozy.contacts.md

```diff
@@ -20,24 +20,28 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
 - `note?`: {string}
 - `email?`: {array} An array of email addresses objects with the following attributes:
   - `address`: {string} Email address
-  - `type?`: {string} Programmatic type of email (`"work"`, `"home"`, `"other"`)
-  - `label?`: {string} A user-provided localized type of email (example: `"Work"`)
+  - `type?`: {string} A user-provided localized type of phone number (example: `"Work"`)
   - `primary?`: {boolean} Indicates a preferred-use address
 - `address?`: {array} An array of postal addresses objects with the following attributes:
-  - `street?`: {string}
-  - `pobox?`: {string}
-  - `city?`: {string}
-  - `region?`: {string}
-  - `postcode?`: {string}
-  - `country?`: {string}
-  - `type?`: {string} Programmatic type of address (`"work"`, `"home"`, `"other"`)
+  - `street?`: {string} Street name
+  - `pobox?`: {string} P.O Box number
+  - `city?`: {string} City name
+  - `region?`: {string} Region name
+  - `number?`: {string} Lane number
+  - `locality?`: {string} Locality name
```

Comment:
What is the difference between city and locality? 

I read that: 
```
the locality (e.g., city)
the region (e.g., state or province);
```

Reply:
C'est une donnée à usage interne, tout comme `building`, `stairs`, `floor` ou `entrycode`, c'est un champs qui sert à renseigner un Lieu-dit

Reply:
OK que dis-tu dans ce cas, de mettre ces infos là dans un sous-objet qu'on nommerait `adress-extended`? Comme ça, on sépare bien ce qui est de la norme, de ce qui est ajouté.

Reply:
J'avais pensé au `ext` du VCard, mais qui convenait pas dans ce cas.
Ok, j'aime bien l'idée, je regarde ça 👍

Reply:
Voila qui est fait, j'ai également mis à jour https://github.com/cozy/cozy-contacts/pull/920 en conséquence